dask/tests/test_base.py

```diff
@@ -983,6 +983,17 @@ def test_persist_scalar_rename():
     dd.utils.assert_eq(dds2, 5)
 
 
+@pytest.mark.skipif("not dd or not np")
```

Comment:
If you put this with the rest of the dataframe groupby tests, then this skip shouldn't be necessary.